apps/www/components/Sections/TwitterSocialProof.tsx

```diff
@@ -53,70 +57,38 @@ function TwitterSocialProof() {
           </div>
         </div>
       </div>
-      <div className="mt-6">
-        <div className="cursor-move lg:-mr-32 lg:-ml-32">
-          <Swiper
-            loop={true}
-            initialSlide={3}
-            spaceBetween={0}
-            slidesPerView={4}
-            speed={300}
-            navigation={{
-              prevEl: prevRef.current,
-              nextEl: nextRef.current,
-              // prevEl: prevRef.current ? prevRef.current : undefined,
-              // nextEl: nextRef.current ? nextRef.current : undefined,
-            }}
-            onInit={(swiper: any) => {
-              swiper.params.navigation.prevEl = prevRef.current
-              swiper.params.navigation.nextEl = nextRef.current
-              // swiper.navigation.update()
-            }}
-            breakpoints={{
-              320: {
-                slidesPerView: 1,
-              },
-              720: {
-                slidesPerView: 2,
-              },
-              920: {
-                slidesPerView: 3,
-              },
-              1024: {
-                slidesPerView: 4,
-              },
-              1208: {
-                slidesPerView: 5,
-              },
-            }}
-          >
-            {Examples.map((tweet: any, i: number) => {
-              return (
-                <SwiperSlide key={i}>
-                  <div className="mr-3 ml-3">
-                    <Link href={tweet.url}>
-                      <a target="_blank" className="cursor-pointer">
-                        <TweetCard
-                          key={i}
-                          handle={`@${tweet.handle}`}
-                          quote={tweet.text}
-                          img_url={`${basePath}${tweet.img_url}`}
-                        />
-                      </a>
-                    </Link>
-                  </div>
-                </SwiperSlide>
-              )
-            })}
-            <div className="container mx-auto mt-3 hidden flex-row justify-between md:flex">
-              <div ref={prevRef} className="p ml-4 cursor-pointer">
-                <IconArrowLeft />
-              </div>
-              <div ref={nextRef} className="p mr-4 cursor-pointer">
-                <IconArrowRight />
-              </div>
+      <div className="lg:-mx-10 xl:-mx-18 mt-6">
+        <div
+          className={`tweet-container columns-1 sm:columns-2 lg:columns-3 xl:columns-4 gap-4 overflow-hidden relative h-[1000px]`}
+        >
+          {showButton && (
+            <div
+              className={`absolute bottom-0 left-0 z-10 w-full h-[25%] bg-gradient-to-t from-[#1c1c1c] via-[#1c1c1c]`}
```

Comment:
You can simply add a margin-bottom: 20px here or use the `bottom` property, something like bottom-4 instead of bottom-0?

Reply:
Yeah, it just depending how much of the gradient you want to hide the tweet cards.